src/client.cpp

```diff
@@ -525,13 +525,10 @@ PVR_ERROR SignalStatus(PVR_SIGNAL_STATUS &signalStatus) { return PVR_ERROR_NO_ER
 PVR_ERROR GetStreamProperties(PVR_STREAM_PROPERTIES* pProperties) { return PVR_ERROR_NOT_IMPLEMENTED; }
 void DemuxAbort(void) { return; }
 DemuxPacket* DemuxRead(void) { return NULL; }
-PVR_ERROR DialogChannelScan(void) { return PVR_ERROR_NOT_IMPLEMENTED; }
 PVR_ERROR CallMenuHook(const PVR_MENUHOOK &menuhook, const PVR_MENUHOOK_DATA &item) { return PVR_ERROR_NOT_IMPLEMENTED; }
```

Comment:
Because those are outdated and updated stubs already exist at https://github.com/hudokkow/pvr.pctv/blob/97a2ff79d377f64fe0281d6de284da77d5c3f5b3/src/client.cpp#L561

This addon didn't existed in xbmc-pvr-addons when you added API 1.9.4 changes. I suppose original addon author added the stubs instead of adapting existing ones. See https://github.com/kodi-pvr/pvr.vuplus/commit/36fbc027e562b5d0cf096ce8a22af462342e4d98 for an example.